aiida/manage/external/postgres.py

```diff
@@ -108,6 +108,8 @@ def create_dbuser(self, dbuser, dbpass, privileges=''):
             self.connection_mode == PostgresConnectionMode.PSYCOPG
         """
         self.execute(_CREATE_USER_COMMAND.format(dbuser, dbpass, privileges))
+        # this is needed on some postgresql installations
```

Comment:
Could you please add more info on what this is doing? It seems quite weird to `GRANT` a user to `current_user`. What does that mean? Grant all rights that `dbuser` has also to `current_user`?

Reply:
I've updated the comment. Let me know whether it is understandable now

Reply:
Cheers.